src/stories/components/HorizontalTitle/styles.scss

```diff
@@ -0,0 +1,45 @@
+@import '../../../scss/variables.scss';
+
+// @file
+// HorizontalTitle
+
+.horizontal-title {
+    font-family: 'Work Sans';
+    padding-top: 0;
+    margin-top: 0;
+
+    &--medium {
+        .horizontal-title__title {
```

Comment:
`h1` font size should be the across across all component's instances. Remove this. What needs to change is the font size of the paragraph text.